docs/release-notes/NuGet-5.1-RTM.md

```diff
@@ -0,0 +1,81 @@
+#5.1 Release Notes
+
+[Full Changelog]()
+
+[Issues List](https://github.com/nuget/home/issues?q=is%3Aissue+is%3Aclosed+milestone%3A%225.1")
+
+**Bug:**
+
+* Plugins:  exception details lost during plugin creation - [#8057](https://github.com/NuGet/Home/issues/8057)
+
+* PackageReference range with exclusive lower bound does not work if the lower bound is present on one of the sources. - [#8054](https://github.com/NuGet/Home/issues/8054)
+
+* Improve IsPackableFalseError message - [#8021](https://github.com/NuGet/Home/issues/8021)
+
+* Packages Lock File - regenerate lock file when project graph changes - [#8019](https://github.com/NuGet/Home/issues/8019)
+
+* ProjectSystem bug: Nuget Packages getting auto removed - [#8017](https://github.com/NuGet/Home/issues/8017)
```

Comment:
Do we want to include something like this in our release notes?

Reply:
i left it in on purpose. my thought was that we are getting many feedback issues about it...
i was hoping this would help people figure out that it was fixed...cause it feels like a nuget issue.
happy to talk through it, if anybody is strongly against it, or wants to tweak it.

Reply:
Not against, just wanted to make sure it's deliberate.